EvtWeightTools/interface/EvtWeightRecorder.h

```diff
@@ -219,13 +221,18 @@ class EvtWeightRecorder
   void
   record_prescale(double weight);
 
+  std::pair<std::vector<int>, std::vector<int>>
+  findHiggs(const LHEParticleCollection & lheParticles, std::vector<int> & exclude);
```

Comment:
I think that this function can be moved under `protected` and made `const` (ie add `const` before the semicolon) or `static` because the function does not touch any member variables.